src/main/java/com/allthemods/gravitas2/GregitasCore.java

```diff
@@ -90,4 +95,29 @@ public void missingMappings(MissingMappingsEvent event) {
         event.getMappings(ForgeRegistries.Keys.ITEMS, "gregitas").forEach(GregitasUtil::remap);
     }
 
+    @SuppressWarnings("resource")
+    @SubscribeEvent
+    public void detectEnteringClaimedChunks(EnteringSection event) {
+        if (!(event.getEntity() instanceof ServerPlayer player)) return;
+        if (!event.didChunkChange()) return;
+        if (!FTBChunksAPI.api().isManagerLoaded()) return;
+        var chunkManager = FTBChunksAPI.api().getManager();
+        var oldChunkDimPos = new ChunkDimPos(player.kjs$getLevel().dimension(), event.getOldPos().chunk());
```

Comment:
Why use the KJS method?

Reply:
cuz i'm a KJS kiddo hehe
actually because it was the first that showed up, no particular reason